### PR TITLE
Feature/quiz step 3

### PR DESCRIPTION
**Pull request by flipkoehler**

### CODE REVIEW

File: components/Quiz/quizLogic.js [hunk 1]
```diff
@@ -1,123 +1,100 @@
...
-            {currentStep < questionAnswer.length - 1
-              ? "Weiter"
-              : "Zur Filmempfehlung"}
-          </button>
-        </div>
-      </section>
-    </form>
+    <QuizSteps
+      currentQuestion={updatedQuestionAnswer}
+      handleNext={handleNext}
```
Comment (julialoeschel) [inline components/Quiz/quizLogic.js:95]: the naming convention would be: 
onNext={handleNext}

File: components/Quiz/QuizSteps.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { movieMood } from "../../assets/data/moods";
+
+export default function QuizSteps({
+  currentQuestion,
+  handleNext,
```
Comment (julialoeschel) [inline components/Quiz/QuizSteps.js:7]: if you follow the naming convention -> onNext

File: components/Quiz/QuizSteps.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        <div>
+          <button
+            disabled={
+              givenAnswers[currentStep].answerOptions.filter(
+                (answer) => answer.checked === true
+              ).length === 0
+                ? true
+                : false
+            }
+            onClick={(event) => handleNext(event, givenAnswers[currentStep])}
```
Comment (julialoeschel) [inline components/Quiz/QuizSteps.js:63]: handleNext -> onNext

**Review by julialoeschel — COMMENTED** [review]
good job 
I have some remarks regarding the naming conventions, other than that - great work!
:octocat:

**Comment by flipkoehler** [comment]
Hey @julialoeschel,
thank you for the good feedback 😊 
I have renamed the two remarks.

**Pull request merged**